### PR TITLE
Message styles

### PR DESCRIPTION
**Pull request by stephywells**
The goal of this PR is to make the messages more consistent, based on the design here:
https://www.figma.com/file/AkQRRLqb8qcf1S5APxyzsG/FF_design_system?node-id=3398%3A30885

@garretlaxton it would be great if you could look over this, and see you you find any pro tips or messages that don't look right.

Before:
Screen Shot 2022-11-04 at 1 12 23 PM
Screen Shot 2022-11-04 at 1 09 04 PM
Screen Shot 2022-11-04 at 1 09 24 PM
Screen Shot 2022-11-04 at 1 11 45 PM
Screen Shot 2022-11-04 at 1 08 43 PM
Screen Shot 2022-11-04 at 1 17 17 PM
Screen Shot 2022-11-04 at 1 18 04 PM

After:
Screen Shot 2022-11-04 at 12 53 43 PM
Screen Shot 2022-11-04 at 12 53 58 PM
Screen Shot 2022-11-04 at 1 34 17 PM
Screen Shot 2022-11-04 at 1 01 30 PM
Screen Shot 2022-11-04 at 1 35 21 PM
Screen Shot 2022-11-04 at 1 31 03 PM
Screen Shot 2022-11-04 at 1 30 25 PM

### CODE REVIEW

**Comment by garretlaxton** [comment]
When adding or deleting a log the notification is the default WP post notification.
image

I haven't seen any other locations.

**Review by Crabcyborg — APPROVED** [review]
Thanks @stephywells!

File: classes/controllers/FrmEntriesController.php [hunk 1]
```diff
@@ -473,7 +473,7 @@ public static function destroy() {
 
 		$message = '';
 		if ( FrmEntry::destroy( $params['id'] ) ) {
-			$message = __( 'Entry was Successfully Deleted', 'formidable' );
+			$message = __( 'Entry was successfully deleted', 'formidable' );
```
Comment (Crabcyborg) [inline classes/controllers/FrmEntriesController.php:476]: @stephywells Should we update this message when you save form settings? I noticed it's all in caps too.

Screen Shot 2022-11-11 at 4 38 26 PM

Comment (stephywells) [reply classes/controllers/FrmEntriesController.php:476]: Sure! It feels weird to me with title case when it's actually a sentence...

**Pull request merged**